src/test/java/io/appium/java_client/android/AndroidAbilityToUseSupplierTest.java

```diff
@@ -37,15 +37,16 @@ public class AndroidAbilityToUseSupplierTest extends BaseAndroidTest {
     };
 
     private final ActionSupplier<AndroidTouchAction> verticalSwiping = () ->
-        new AndroidTouchAction(driver)
-                .press(element(driver.findElement(AppiumBy.accessibilityId("Gallery"))))
+            new AndroidTouchAction(driver)
+                    .press(element(driver.findElement(AppiumBy.accessibilityId("Gallery"))))
 
-                .waitAction(waitOptions(ofSeconds(2)))
+                    .waitAction(waitOptions(ofSeconds(2)))
 
-                .moveTo(element(driver.findElement(AppiumBy.accessibilityId("Auto Complete"))))
-                .release();
+                    .moveTo(element(driver.findElement(AppiumBy.accessibilityId("Auto Complete"))))
+                    .release();
```

Comment:
please revert changes in formatting, such changes bring a lot of noise and make review process harder

Reply:
@valfirst I agree, could you please let me know if there is any code formatter that i can use here?

Reply:
try https://plugins.jetbrains.com/plugin/8527-google-java-format

Reply:
Fixed Formatter